### PR TITLE
Fixes #12919 - allows mass assignment of provisioning_template from os

### PR DESCRIPTION
**Pull request by orrabin**

### CODE REVIEW

File: app/models/operatingsystem.rb [hunk 1]
```diff
@@ -12,7 +12,7 @@ class Operatingsystem < ActiveRecord::Base
       :architectures, :architecture_ids, :architecture_names,
       :medium_ids, :medium_names,
       :os_default_templates_attributes, :os_parameters_attributes,
-      :ptable_ids, :ptable_names
+      :ptable_ids, :ptable_names, :provisioning_templates, :provisioning_template_id
```
Comment (ares) [inline app/models/operatingsystem.rb]: I think it should be :provisioning_template_id**s**

Comment (orrabin) [inline app/models/operatingsystem.rb]: thanks, I'll change it

**Comment by orrabin** [comment]
@ares fixed your comment

File: app/models/operatingsystem.rb [hunk 2]
```diff
@@ -12,7 +12,7 @@ class Operatingsystem < ActiveRecord::Base
       :architectures, :architecture_ids, :architecture_names,
       :medium_ids, :medium_names,
       :os_default_templates_attributes, :os_parameters_attributes,
-      :ptable_ids, :ptable_names
+      :ptable_ids, :ptable_names, :provisioning_templates, :provisioning_template_ids
```
Comment (domcleal) [inline app/models/operatingsystem.rb]: all _ids attributes should have the corresponding _names too.

I think the line should be moved up too, it looks like these associations are in alphabetical order and roughly one association per line.

**Comment by orrabin** [comment]
@domcleal I added the _names and moved the association to it's own line

**Comment by domcleal** [comment]
Merged as 99bb178, thanks @orrabin.

**Pull request closed without merging**